modules/ROOT/pages/_partials/include-traffic-inspection.adoc

```diff
@@ -0,0 +1,90 @@
+//end::traffic-inspection-overview[]
+//
+//
+//tag::traffic-inspection-limitations[]
+
+At this time, the Anypoint Monitoring agent that supports traffic inspection is available for Mule instances running on _Linux_. 
+//end::traffic-inspection-limitations[]
+//
+//
+//
+//tag::traffic-inspection-prereqs[]
+
+**  A closed beta version of the Anypoint Monitoring agent from a Salesforce representative
++
+The agent's zip file is available to a limited customer base.
+** A Mule server that is registered in Runtime Manager 
++
+To set up a Mule server, see xref:runtime-manager::rtm-traffic-inspection.adoc[].
+** The private certificate in the proxy that communicates with the Anypont Monitoring agent in the Mule runtime instance includes `us1.ingest.mulesoft.com` in its Subject Alternative Name (SAN).
++
+In addition, you must add this private certificate or its CA certificate to the operating system's system-wide truststore. For guidance, see https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/securing_networks/using-shared-system-certificates_securing-networks#adding-new-certificates_using-shared-system-certificates[Adding new certificates] in the Redhat Linux documentation.
+** Access to Anypoint Monitoring 
++
+An Anypoint Platform administrator can provide access from Access Manager, in Anypoint Platform. For more information, see xref:monitoring::am-permissions.adoc[].
+
+//** The Runtime Manager Agent must be installed. See xref:runtime-manager::runtime-manager-agent.adoc[].
+// Perhaps if server is set up, it would be installed? Without it, I got this error: 
+// NoSuchFileException: /Users/sduke/Downloads/mule-enterprise-standalone-4.8.0/conf/mule-agent.yml
+//end::traffic-inspection-prereqs[]
+//
+
+
+//
+//
+//tag::traffic-inspection-procedure[]
+
+To enable traffic inspection: 
+
+. If an Anypoint Monitoring agent is installed on your server, uninstall the agent and delete its `/am` subfolder:
+
+.. From your Mule home directory (`MULE_HOME`), run the following command to uninstall the agent:
++
+----
+./bin/uninstall 
+----
+.. Delete the `/am` subfolder from your Mule home directory.
+. Place the Anypoint Monitoring agent zip file (`am-3.x.zip`) into your Mule home directory (`MULE_HOME`).
++
+The zip is available as part of the closed beta program. 
+. Extract the zip file to add the `/am` subfolder into your Mule home directory (`MULE_HOME`). 
+. From the command line:
+.. Navigate to the `/am` subfolder, for example:
++
+----
+cd am
+----
+.. Install the agent to your Linux server:
++
+----
+./bin/install -p http:/proxy-server:port
+----
++
+Replace `+http:/proxy-server:port+` with the URI of your proxy server and port.
+. Restart your server in Runtime Manager.
++
+For guidance, see xref:runtime-manager::servers-actions.adoc[]
+. Return to your `/am` subfolder and run the setup script:
```

Comment:
I think we can remove this line as the use is already in the `am` folder.

Reply:
```suggestion
. From your `am` subfolder, run the setup script:
```